crypto/blake2/TODO

```diff
@@ -0,0 +1,4 @@
+The code uses NATIVE_LITTLE_ENDIAN in #if statements.  Is this right?
```

Comment:
I don't think we have any such define.  But we have L_ENDIAN and B_ENDIAN.  They might not be defined, but if they're defined you can use it optimize things knowing the endianness.  I guess you just have to replace all your cases with `#ifdef L_ENDIAN`